src/Host.h

```diff
@@ -380,6 +382,8 @@ class Host : public QObject
 
     cTelnet mTelnet;
     QPointer<TMainConsole> mpConsole;
+    dlgPackageManager* mpPackageManager;
```

Comment:
Could you use a `std::unique_ptr` here instead of the raw pointer? It's the modern C++ way and it helps avoid an accidental memory leak when the object goes out of scope.

Reply:
> Could you use a `std::unique_ptr` here instead of the raw pointer? It's the modern C++ way and it helps avoid an accidental memory leak when the object goes out of scope.

I tried and failed. It added a lot of complexity and had undefined behaviour in the end (crashes). 
I've never used std::unique_ptr before.
Shouldn't ```setAttribute(Qt::WA_DeleteOnClose);``` be enough to avoid memory leaks as it should delete the object if it is closed?

Maybe someone with more experience should add this change (if needed).

Reply:
OK, let's do this later Add your name to the copyright and it's good.

Reply:
> OK, let's do this later Add your name to the copyright and it's good.

Done.
Was a bit hesitant as I wasn't sure if I would also have to find out who the other contributors were as most of it is just copy/paste :wink:

Reply:
Yep good point. I've added the original authors. @SlySven could you add yourself too where you see fit?